packages/core/src/utils/camelCaseConversion.ts

```diff
@@ -0,0 +1,13 @@
+export function camelCaseToSentenceCase(str: string) {
```

Comment:
I'd hoped to make this generally available but we have some circular dependencies